signer/src/config/mod.rs

```diff
@@ -740,6 +782,7 @@ mod tests {
         remove_parameter("signer_round_max_duration");
         remove_parameter("bitcoin_presign_request_max_duration");
         remove_parameter("dkg_max_duration");
+        remove_parameter("dkg_max_duration");
```

Comment:
nit?
```suggestion
        remove_parameter("max_deposits_per_bitcoin_tx");
```

Reply:
lol, oops. 🙏🏽

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1181/commits/29a3f839d3dcbd53856fa73d8d7da0742da00328 (with a typo in the git commit message 😄)